File: app/src/main/java/com/returnnotfound/stain/utils/AppUtils.java
package com.returnnotfound.stain.utils;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;

public abstract class AppUtils {
  public static void openAppInStore(Context context) {
    final String appPackageName = context.getPackageName();
    try {
      context.startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("market://details?id=" + appPackageName)));
    } catch (android.content.ActivityNotFoundException e) {
      context.startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://play.google.com/store/apps/details?id=" + appPackageName)));
    }
  }
}

File: app/src/main/java/com/returnnotfound/stain/base/utils/AppUtils.kt
package com.returnnotfound.stain.base.utils

import android.content.ActivityNotFoundException
import android.content.Context
import android.content.Intent
import android.net.Uri

object AppUtils {

  @JvmStatic
  fun openAppInStore(context: Context) {
    val appPackageName = context.packageName
    try {
      context.startActivity(
        Intent(
          Intent.ACTION_VIEW,
          Uri.parse("market://details?id=$appPackageName")
        )
      )
    } catch (e: ActivityNotFoundException) {
      context.startActivity(
        Intent(
          Intent.ACTION_VIEW,
          Uri.parse("https://play.google.com/store/apps/details?id=$appPackageName")
        )
      )
    }
  }
}